### ISSUE TITLE
Ctrl-V terminates program

### ISSUE THREAD

**Issue by amutech1008** [gaphor/gaphor#1814]
### Describe the bug
Gaphor closes without warnings and without saving data when the Ctrl-V key combination is used to paste text into element names.

### To Reproduce
1. Go to a SysML Block diagram
2. Add a Block element
3. Deselect the Block element
4. Go another program and Ctrl-C to copy some text
5. Double click on the Block element in Gaphor
6. Press Ctrl-V to paste the text into the name field of the Block
7. Observe that Gaphor terminates without warning and without saving the data.

If possible, add the error report from `≡` (top-right menu) → `Tools` → `Error Reports`.

### Expected behavior
Gaphor does not terminate unexpectedly but pastes the text into the name field.

### Screenshots
Not possible to get a screenshot becuase the program terminates without warnings.

### OS
- [ ] Linux (Please put in notes the specific distro)
- [ ] macOS
- [ X] Windows

### Version
Version of Gaphor: 2.12.1

### Additional information
None

**Comment by danyeaw** [comment]
Hi @amutech1008 Thanks for the bug report. Unfortunately, I am not able to reproduce this in Windows or Linux. Which version of Windows are you running?

**Comment by amolenaar** [comment]
@amutech1008 What program did you copy from?

**Comment by amutech1008** [comment]
@amolenaar Notepad, Notepad++, and all other programs that I have tried. And it is not due to special chars, because even pasting a single letter causes Gaphor to close /Allan

**Comment by vanillajonathan** [comment]
Run Gaphor from the terminal, this way when the application crashes you can see the error output in the terminal.

**Comment by amutech1008** [comment]
Both CMD and PowerShell just launches a session with Gaphor (as i a new thread). So there is no output in the terminal.

**Comment by amutech1008** [comment]
Thanks for the comments above. Setting up the developement environment and participating in the bug fixing is outside my scope of using Gaphor. You can close the case as not planned. Best regards, Allan.

**Comment by amolenaar** [comment]
Has this issue been resolved with the latest update of gvs-build?

**Comment by jacobgrass** [comment]
@danyeaw This issue is present in version 2.19.3 on Windows 10.

**Comment by danyeaw** [comment]
@jacobgrass yup, we had to revert our GTK version due to another issue on Windows with icons. We are getting ready to release 2.20.0 and it should be resolved there.
